tests/unit/uncertainty/test_predictors.py

```diff
@@ -0,0 +1,65 @@
+from lightning import pytorch as pl
+import pytest
+import torch
+from torch.utils.data import DataLoader
+
+from chemprop.data import MoleculeDatapoint, MoleculeDataset, collate_batch
+from chemprop.models import MPNN
+from chemprop.uncertainty.predictor import DropoutPredictor, EnsemblePredictor
+
+
+@pytest.fixture
+def dataloader(mol_regression_data):
+    smis, Y = mol_regression_data
+    data = [MoleculeDatapoint.from_smi(smi, y) for smi, y in zip(smis[:2], Y[:2])]
```

Comment:
```suggestion
    data = [MoleculeDatapoint.from_smi(smi, y) for smi, y in zip(smis, Y)]
```

Reply:
Since the prediction results are tested as well, it makes sense to me to only use a small set of data.